common/src/main/java/org/vivecraft/api/data/VRPose.java

```diff
@@ -0,0 +1,63 @@
+package org.vivecraft.api.data;
+
+import com.google.common.annotations.Beta;
+import net.minecraft.world.phys.Vec3;
+import org.joml.Quaternionf;
+
+/**
+ * Represents the pose data, such as position and rotation, for a given trackable object, such as the HMD or
+ * a controller.
+ */
+public interface VRPose {
+
+    /**
+     * @return The position of the device in Minecraft world coordinates.
+     */
+    Vec3 getPos();
+
+    /**
+     * @return The rotation of the device.
+     */
+    Vec3 getRot();
+
+    /**
+     * @return The pitch of the device in radians.
+     */
+    double getPitchRad();
+
+    /**
+     * @return The yaw of the device in radians.
+     */
+    double getYawRad();
+
+    /**
+     * @return The roll of the device in radians.
+     */
+    default double getRollRad() {
```

Comment:
Same here, do we need them? We should try to either use radians or degrees everywhere in the code.

Reply:
I can do my best to get it all to be in either radians or degrees so that the programmer can just convert to the other when needed. Which do you think would be better to expose?

Reply:
It's all radians now as of https://github.com/hammy275/VivecraftMod/commit/adc5a17fd240c071d2fbedc544a6fbc1ba1ecb92 !